CHANGELOG.md

```diff
@@ -6,6 +6,42 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 1.0.0
+
+This release upgrades to Stylelint 15 and now includes:
+
+* Checks for media query syntax, rules and values
+* Checks for number precision inside CSS functions
+* Fixes for duplicate rules with intentionally different units
+
+Read about breaking changes in the Stylelint [**Migrating to v15.0.0**](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md#breaking-changes) guide.
+
+### Support for Internet Explorer 8, 9 and 10
+
+Support for IE8–10 has not been a requirement for services since June 2018, so we removed the rule [`selector-pseudo-element-colon-notation`](https://stylelint.io/user-guide/rules/selector-pseudo-element-colon-notation/) preventing double-colon `::before` and `::after` pseudo element notation.
+
+This change was made in [pull request #36: Remove `selector-pseudo-element-colon-notation`](https://github.com/alphagov/stylelint-config-gds/pull/36).
+
+### Checking for invalid property/value pairs
+
+Stylelint 15 adds an optional new rule [`declaration-property-value-no-unknown`](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown) to check for invalid property/value combinations such as:
+
+```css
+.govuk-link--example {
+  top: red;
+}
+```
+
+Do try it on your projects before we enable it in future, but [report any problems on this issue](https://github.com/alphagov/stylelint-config-gds/issues/42) to help us assess its impact.
+
+### Deprecated stylistic rules
+
+As mentioned in the [`stylelint-config-gds@0.3.0` release notes](#030) below, this release [deprecates all stylistic rules](https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules). We no longer include rules that would format your styles and recommend you use [Prettier](https://prettier.io/) for formatting instead.
+
+>When we created these rules, pretty printers (like Prettier) didn't exist. They now offer a better way to consistently format code, especially whitespace. Linters and pretty printers are complementary tools that work together to help you write consistent and error-free code.
+
+If this is not possible, you can configure the [`stylelint-stylistic`](https://github.com/elirasza/stylelint-stylistic) or [`stylelint-codeguide`](https://github.com/firefoxic/stylelint-codeguide) plugins to restore the [deprecated rules](https://github.com/alphagov/stylelint-config-gds/pull/32/commits/0bab4e7596d642c6fa65bc133e24968a4ff74671) instead.
```

Comment:
I think this note needs to be more prominent and more significant, by labelling this as "Deprecated stylistic rules" it doesn't sound like they're gone but they have and it's a very impactful for users of this tooling.

I'd suggest we up the prominence of this to make this the top level item in the changelog, with something like:

> ## Breaking change: Removal of stylistic rules
> 
> This release no longer has stylistic rules (such as tabs/spaces, indentation, etc) as these have been removed from Stylelint common configurations as part of their [deprecation in Stylelint 15]((https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules)).
> 
> As per Stylelint's own documentation, we recommend that projects adopt [Prettier](https://prettier.io/) for formatting instead.
> 
> If this is not possible for your project, you can [configure](https://stylelint.io/user-guide/configure/) your projects' stylelint configuration to use the [`stylelint-stylistic`](https://github.com/elirasza/stylelint-stylistic) or [`stylelint-codeguide`](https://github.com/firefoxic/stylelint-codeguide) plugins to restore the [deprecated rules](https://github.com/alphagov/stylelint-config-gds/commit/c23cf01d5a2c60208649b66665809801f9c3a915) instead.
> 

(note I've updated the link to the commit because the previous one broke on a rebase. If we rebase again that one will still work but be detached so we should be careful to try avoid further rebasing without also updating this)

Reply:
Good point regarding the rebase, thanks @kevindew

I'll split the rules removal out into a separate PR so we don't lose it, good for project history

Agree on the prominence thing, I'll do another update shortly. Appreciate it

Reply:
How's this? I've updated the description and moved the rule changes to another PR

* https://github.com/alphagov/stylelint-config-gds/pull/44

We can then link to the PR in the https://github.com/alphagov/stylelint-config-gds/pull/44#pullrequestreview-1539193493 CHANGELOG entry